api/serviceEntries/serviceEntriesModel.js

```diff
@@ -92,7 +94,7 @@ const findById = async (id) => {
 
 // Can only update service_entries data in table on front end
 // Can't update "extra" data that's tacked onto the responses
-// Can't edit: first_name, last_name, service_type_name, status_name
+// Can't edit: first_name, last_name, service_type_name, status_name _
```

Comment:
Hey, think this was just a typo, non-breaking, but may want to get rid of that before you merge